examples/basic.php

```diff
@@ -3,7 +3,9 @@
 require __DIR__ . '/../vendor/autoload.php';
 
 Amp\Loop::run(static function () {
-    $client = new Amp\Redis\Redis(new Amp\Redis\RemoteExecutor('tcp://localhost:6379'));
+    $client = new Amp\Redis\Redis(new Amp\Redis\RemoteExecutor(
+        new Amp\Redis\Config('tcp://localhost:6379')
```

Comment:
This should be `Config::fromUri`, as the constructor has been made private in `master`.